pages/dao/[symbol]/proposal/new.tsx

```diff
@@ -216,6 +237,53 @@ const New = () => {
     setFormErrors({})
     setForm({ ...form, [propertyName]: value })
   }
+
```

Comment:
i'd prefer it if this stuff was more separated, this component is too big. we can chat about the best way to accomplish that if you like.